python/torch_mlir_e2e_test/test_suite/constant_alloc.py

```diff
@@ -215,6 +215,186 @@ def OnesModuleCPUDevice_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class AtenEyeModuleDefaultDtype(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+    ])
+    def forward(self):
+        return torch.ops.aten.eye(3)
+
+
+@register_test_case(module_factory=lambda: AtenEyeModuleDefaultDtype())
+def AtenEyeModuleDefaultDtype_basic(module, tu: TestUtils):
+    module.forward()
+
+
+class AtenEyeModuleInt2D(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+    ])
+    def forward(self):
+        return torch.eye(3, dtype=torch.int64)
+
+
+@register_test_case(module_factory=lambda: AtenEyeModuleInt2D())
+def AtenEyeModuleInt2D_basic(module, tu: TestUtils):
+    module.forward()
+
+
+class AtenEyeModuleFloat2D(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+    ])
+    def forward(self):
+        return torch.eye(3, dtype=torch.float32)
+
+
+@register_test_case(module_factory=lambda: AtenEyeModuleFloat2D())
+def AtenEyeModuleFloat2D_basic(module, tu: TestUtils):
+    module.forward()
+
+class AtenEyeModuleFalsePinMemory(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+    ])
+    def forward(self):
+        return torch.ones(3, dtype=torch.float32, pin_memory=False)
```

Comment:
there's a couple of tests that return `torch.ones` rather than `torch.eye`

Reply:
fixed